model_lab_configs/huggingface/Intel/bert-base-uncased-mrpc/1/bert_trtrtx.json.config

```diff
@@ -0,0 +1,95 @@
+{
+    "name": "Convert to NVIDIA GPU (TRT RTX)",
```

Comment:
We would like to call it `Convert to NVIDIA TRT for RTX` https://developer.nvidia.com/tensorrt-rtx